lib/normalizers/v1.js

```diff
@@ -41,15 +52,26 @@ _.assign(Builders.prototype, {
      */
     auth: function (entityV1) {
         if (!entityV1) { return; }
+        // Handle the noauth/normal case for new and legacy attributes first, in that order
         if (util.notLegacy(entityV1, 'auth') && entityV1.auth) {
             return (entityV1.auth.type === 'noauth') ? { type: 'noauth' } : util.sanitizeAuthArray(entityV1);
         }
-        if (!entityV1.currentHelper) { return; }
+        if (entityV1.currentHelper === 'normal') { return { type: 'noauth' }; }
+
+        var auth,
+            params,
+            mapper,
+            currentHelper = entityV1.currentHelper,
+            helperAttributes = entityV1.helperAttributes;
+
+        if (!(currentHelper || this.options.noDefaults) || (currentHelper && !helperAttributes)) { return; }
```

Comment:
Can you add comments explaining the `noDefaults` option?

Reply:
Done. 👍